app/[id]+api.ts

```diff
@@ -1,79 +1,86 @@
-import { deleteGuest, getGuest, Guest } from '../database/guests';
+import { deleteGuestInsecure, getGuestInsecure } from '../database/guests';
+import { ExpoApiResponse } from '../ExpoApiResponse';
+import { Guest, guestsSchema } from '../migrations/00000-createTableGuests';
+
+type ExpoResponseBodyGet =
+  | {
+      guest: Guest;
+    }
+  | { error: string };
```

Comment:
nit: these objects should be the same code style (either both on a single line, or both on multiple lines)

(same for any other API Routes)

Reply:
Fixed format, both single line
[ce6c4f6](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/4/commits/ce6c4f6d7dd5ef51784cc65fa041ebdc1adb47cd)